python/src/table.rs

```diff
@@ -86,10 +86,10 @@ impl Table {
         })?)
     }
 
-    fn add_parquet<'py>(&mut self, py: Python<'py>, path: String) -> Result<&'py PyAny> {
+    fn add_parquet<'py>(&mut self, py: Python<'py>, url: String) -> Result<&'py PyAny> {
```

Comment:
Want to go back to `path`, if we'll continue to support relative / absolute non-url paths?

Reply:
Don't think so -- it's more like a URL (or an `href`)?